apps/projects/app/components/Panel/ReviewApplication/ReviewApplication.js

```diff
@@ -22,245 +21,185 @@ import { ipfsAdd } from '../../../utils/ipfs-helpers'
-      applicationDate: request.applicationDate
-    }
-
-    const applicant = application.user
-    const applicantName = applicant.name ? applicant.name : applicant.login
-    const applicationDateDistance = formatDistance(new Date(application.applicationDate), new Date())
-    return (
-      <div>
-        <IssueTitle>{issue.title}</IssueTitle>
-
-        <SafeLink
-          href={issue.url}
-          target="_blank"
-          style={{ textDecoration: 'none', color: '#21AAE7' }}
-        >
-          <IssueLinkRow>
-            <IconGitHub color="#21AAE7" width='14px' height='14px' />
-            <Text style={{ marginLeft: '6px' }}>{issue.repo} #{issue.number}</Text>
-          </IssueLinkRow>
-        </SafeLink>
-
-        <FieldTitle>Applicant</FieldTitle>
-        <DropDown
-          name="Applicant"
-          items={issue.requestsData.map( request => request.user.login)}
-          onChange={this.changeRequest}
-          active={this.state.requestIndex}
-          wide
-        />
+  const applicant = application.user
+  const applicantName = applicant.name ? applicant.name : applicant.login
+  const applicationDateDistance = formatDistance(new Date(application.applicationDate), new Date())
 
-        <ApplicationDetails>
-          <UserLink>
-            <img
-              alt=""
-              src={applicant.avatar}
-              style={{
-                width: '32px',
-                height: '32px',
-                marginRight: '10px',
-              }}
-            />
-            <SafeLink
-              href={applicant.url}
-              target="_blank"
-              style={{ textDecoration: 'none', color: '#21AAE7', marginRight: '6px' }}
-            >
-              {applicantName}
-            </SafeLink>
+  return (
+    <div css={`margin: ${2 * GU}px 0`}>
+      <Text.Block size="xlarge">{issue.title}</Text.Block>
+
+      <Link
+        href={issue.url}
+        target="_blank"
+        style={{ textDecoration: 'none', color: `${theme.link}` }}
+      >
+        <IssueLinkRow>
+          <IconGitHub color={`${theme.link}`} width='14px' height='14px' />
+          <Text css="margin-left: 6px">
+            {issue.repo} #{issue.number}
+          </Text>
+        </IssueLinkRow>
+      </Link>
+
+      <FieldTitle>Applicant</FieldTitle>
+      <DropDown
+        name="Applicant"
+        items={issue.requestsData.map( request => request.user.login)}
+        onChange={changeRequest}
+        selected={requestIndex}
+        wide
+      />
+
+      <ApplicationDetails background={`${theme.background}`} border={`${theme.border}`}>
+        <UserLink>
+          <img
+            alt=""
+            src={applicant.avatar}
+            css="width: 32px; height: 32px; margin-right: 10px"
+          />
+          <Link
+            href={applicant.url}
+            target="_blank"
+            style={{ textDecoration: 'none', color: `${theme.link}`, marginRight: '6px' }}
```

Comment:
```js
color: theme.link, marginRight: 6
```

No need for the interpolation; if you give a number to a JSX `style` property, the `px` is implied.

Reply:
Useful to know about `style`! Fixed.